### ISSUE TITLE
Feedback csahlholdt

### ISSUE THREAD

**Issue by csahlholdt** [teokem/project-work-missmagnum#1]
Here are my comments on your notebook organised by the requirements:
- *Include rich documentation using Markdown, equations, tables, links, etc.*
Good use of Markdown with equations, text formatting, images, and links to relevant background.
A little rough around the edges; there is still a personal reminder under "plotting the latent space".
A minor comment: When referring to variables in the code it looks nice with backticks e.g. `True`.
Although it is not a requirement, the code could use a few more comments.

- *Import or generate data. If generating, data should be exported to disk.*
Data imported from keras.datasets.mnist.
Under "Training the model", the plots are only produced if `train = True` which is not the default due to the long training time. It would have been nice if this data was imported and shown when `train = False`.

- *Perform data operations using numpy, scipy, pandas or equivalent.*
Good use of `numpy` and `keras` for data operations.

- *Create plots of publication ready quality.*
The first plots are clear and easy to read.
The final plots from plot_latent would benefit from a discrete colorbar since the values are discrete.
I would be nice with a more concrete explanation in README.md of how to make the plots publication ready instead of 3D. Perhaps an option could be implemented in the notebook to choose between 3D interactive or publication plots.

- *Include instructions on how to run the notebook, include the required packages. This could be an environment.yml file.*
Good instructions under "User Guide" in `README.md`.

- *Be reproducible, i.e. someone else should be able to redo the steps.*
I was able to redo all steps including generating a new mask and retraining the model.
A suggestion: To make sure the results are exactly reproducible, make some default data which is not overwritten if the user chooses to make a new mask or retrain the model. Then that data is always available.

**Comment by missmagnum** [comment]
@csahlholdt Thanks!
